_scripts/release_component.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -euo pipefail
+
+NEW_RELEASE="${NEW_RELEASE:-unknown}"
+DRY_RUN_CMD='echo'
+DRY_RUN=YES
+UPDATE_MASTER=YES
+
+# flags are --help --no-dry-run --no-update-master
+for i in "$@"; do
```

Comment:
I was just obeying my master, `shellcheck`:
```bash
In _scripts/release_component.sh line 11:
for i in $@; do
         ^-- SC2068: Double quote array expansions to avoid re-splitting elements.
```
And in practice, it does handle multiple flags fine:
```bash
$ release_component.sh --no-update-master --no-dry-run
```